filter-api/src/main/java/org/hypertrace/agent/filter/MultiFilter.java

```diff
@@ -48,16 +49,16 @@ public boolean evaluateRequestHeaders(Span span, Map<String, String> headers) {
             t);
       }
     }
-    return shouldBlock;
+    return new FilterResult(false, 403, "Hypertrace Blocked Request");
```

Comment:
the default message and status code would be 200 and OK right? since we 're setting shouldBlock to false

Reply:
yes its a nit only, I get that we would be overriding this in our filter implementation and incase of pass through it wont be set at all, but just for consistency

Reply:
Updated to empty msg and `0` as status code since they should not be used